websub-ballerina/build.gradle

```diff
@@ -365,22 +368,29 @@ publishing {
     }
 }
 
-ballerinaBuild.dependsOn ":${packageName}-native:build"
-ballerinaBuild.dependsOn updateTomlFile
-ballerinaBuild.dependsOn initializeVariables
-ballerinaBuild.dependsOn test
+unpackJballerinaTools.dependsOn copyToLib
```

Comment:
Is there a task named `copyToLib`? Shall we revisit these names?

Reply:
There is a task for `copyToLib` in line 198.